.github/workflows/ui-release-trigger.yml

```diff
@@ -0,0 +1,20 @@
+name: "Trigger Boundary UI release"
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  trigger-circleci-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+      - name: Trigger Release job
+        env:
+          TAG: ${{ github.event.release.tag_name }}
+        run: |
```

Comment:
Should we do any validation on the tag name before attempting a release?  Should we require it look like a semver version?

Reply:
Tagging @99 for visibility.  I agree with @shore.  For the time being let's use what's here.  We'd like to push out a release today.